docs/tables/onfleet_hubs.md

```diff
@@ -0,0 +1,20 @@
+# Table: onfleet_hubs
+
+https://docs.onfleet.com/reference/list-hubs
+
+The composite primary key for this table is (**organization_id**, **id**).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|organization_id (PK)|String|
```

Comment:
Hi James,
Indeed the plugin is only capable of syncing from 1 onfleet account.
I included this just in case a user does sync data from 2 onfleet accounts into one database - to avoid primary-key conflicts in different accounts.
I guess it's a convention in cloudquery to include this info where possible (because in many of our plugins are used in many accounts (e.g. AWS, GCP, ...)).